### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by mhaoli**
* Fix one typo.
* Delete a TODO. This TODO has been addressed in PR #1158.

### CODE REVIEW

File: elasticdl/python/master/servicer.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def __init__(
         if evaluation_service:
             evaluation_service.set_master_servicer(self)
 
-    # TODO: This is currently being used by multiple tests to initilize
+    # TODO: This is currently being used by multiple tests to initialize
```
Comment (chunyang-wen) [inline elasticdl/python/master/servicer.py]: Who is `This`?

Comment (mhaoli) [reply elasticdl/python/master/servicer.py]: `This` refers to function `set_model_var`. I am modifying this comment.

File: elasticdl/python/master/servicer.py [hunk 2]
```diff
@@ -65,8 +65,9 @@ def __init__(
         if evaluation_service:
             evaluation_service.set_master_servicer(self)
 
-    # TODO: This is currently being used by multiple tests to initilize
-    # self._model, where the initialization should be done via constructor.
+    # TODO: Multiple tests are currently using function `set_model_var` to
+    # initialize self._model, where the initialization should be done via
+    # constructor.
```
Comment (chunyang-wen) [inline elasticdl/python/master/servicer.py]: I think there are missing `a`s before the word `function` and `constructor`.

Comment (chunyang-wen) [reply elasticdl/python/master/servicer.py]: What is the `constructor`?

Comment (mhaoli) [reply elasticdl/python/master/servicer.py]: Done.

**Review by chunyang-wen — APPROVED** [review]

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**